### PR TITLE
Update build tools and remove old references to odk

### PR DESCRIPTION
**Pull request by wbrunette**
… for shared-src dir

What is included in this PR?
- Update build tools and environment
- Remove old references and usage of odk
- Remove the  shared-src dir

### CODE REVIEW

File: odkx-src/conf.py [hunk 1]
```diff
@@ -153,7 +153,7 @@
     'pointsize': '12pt',
 
     # Additional stuff for the LaTeX preamble.
-    'preamble': '''
+    'preamble': r'''
```
Comment (jbeorse) [inline odkx-src/conf.py:156]: What is this 'r' for?

File: odkx-src/docs-tech-guide.rst [hunk 2]
```diff
@@ -1053,15 +1053,10 @@ Working on the docs
    Finally, you can open an :ref:`editor of your choice <choose-editor>`
    and work on the documentation.
 
-   The source files for documentation text are in these directories:
+   The source files for documentation text are in these directory:
```
Comment (jbeorse) [inline odkx-src/docs-tech-guide.rst:1056]:
```suggestion
   The source files for documentation text are in this directory:
```

File: odkx-src/conf.py (hunk 1 above)
Comment (wbrunette) [reply odkx-src/conf.py:156]: To not use the unicode escape. We end up compounding problems when trying to escape characters.

Comment (wbrunette) [reply odkx-src/conf.py:156]: It seems the '\u' was producing an error and the 'r' preceding the string turns off the eight-character Unicode escape

**Review by jbeorse — APPROVED** [review]

**Pull request merged**